utils/crunchbase.js

```diff
@@ -113,7 +134,7 @@ export async function createInvestor(investor) {
       elo: 1000,
       name: investor.name,
       numComparisons: 0,
-      image: investor?.image_url
+      image: investor?.image_url ?? "",
```

Comment:
Before releasing the new rankings, make sure to run the script that populates images!